### DISCUSSION TITLE
Task07 Step 2 - Image Path Issues

### DISCUSSION THREAD

**Discussion by SMRazaRizvi96** [chulminy/CIVE497-CIVE700#172] [Q&A] [answered]
Hello All,

I'm facing an issue while running the Step 2 Cell where it joins the folder path to the image path at various places.

 `FileNotFoundError: [Errno 2] No such file or directory: 'img\\img\\IMG_3729.JPG'`

I resolved this issue by changing line 43 inside `mysfm_fun_set.py` to

```
img_file_path = parts[0]
img_file_name = img_file_path.split('\\', 1)[1]
```

I hope it helps

**Comment by chulminy** [comment, marked as answer]
Yes. It depends on where you run the software. That's a good information!

Chul Min

**Comment by hastishwan-uw** [comment]
I have created uimg/img and img/img, and it works ! 👍 
- Just double-checking ?!!! 
[ imgFolder = 'img'  # folder of original input images ] are images that used for the software (VisualSFM)
[ uimgFolder = 'uimg'  # folder of undistorted images ]are images before using it in the software
